crates/searchers/openapi.yaml

```diff
@@ -0,0 +1,584 @@
+
+    Liquidity:
+      description: |
+        On-chain liquidity that can be used in a solution. This liquidity is
+        provided to facilitate TODO
+      allOf:
+        - $ref: "#/components/schemas/LiquidityParameters"
+        - type: object
+          properties:
+            address:
+              $ref: "#/components/schemas/Address"
+            gasEstimate:
+              type: integer
+              description: |
+                A rough approximation in the number gas units required to
+                use this liquidity on-chain.
+
+    Auction:
+      description: |
+        The abstract auction instance in order to solve.
+      type: object
+      properties:
+        id:
+          type: integer
+          description: |
+            The identifier for the auction. Will be set to `null` for requests
+            that are not part of an auction (when quoting token prices for
+            example).
+        tokens:
+          type: object
+          additionalProperties:
+            $ref: "#/components/schemas/Token"
+          description: |
+            A map of token addresses to token information.
+        orders:
+          type: array
+          items:
+            $ref: "#/components/schemas/Order"
+          description: |
+            The solvable orders included in the auction.
+        liquidity:
+          type: array
+          items:
+            $ref: "#/components/schemas/Liquidity"
+          description: |
+            On-chain liquidity that can be used by the solution.
+        gasPrice:
+          $ref: "#/components/schemas/TokenAmount"
+          description: |
+            The current estimated gas price. This is the price that will be used
+            for solution scoring.
+
+    JitOrder:
+      description: |
+        A just-in-time liquidity order included in a settlement.
+      type: object
+      properties:
+        sellToken:
+          $ref: "#/components/schemas/Token"
+        buyToken:
+          $ref: "#/components/schemas/Token"
+        receiver:
+          $ref: "#/components/schemas/Address"
+        sellAmount:
+          $ref: "#/components/schemas/TokenAmount"
+        buyAmount:
+          $ref: "#/components/schemas/TokenAmount"
+        validTo:
+          description: |
+            Unix timestamp until the order is valid to.
+          type: integer
+        appData:
+          $ref: "#/components/schemas/AppData"
+        feeAmount:
+          $ref: "#/components/schemas/TokenAmount"
+        kind:
+          $ref: "#/components/schemas/OrderSide"
+        partiallyFillable:
+          type: boolean
+        sellTokenBalance:
+          $ref: "#/components/schemas/SellTokenSource"
+        buyTokenBalance:
+          $ref: "#/components/schemas/BuyTokenDestination"
+        signingScheme:
+          $ref: "#/components/schemas/SigningScheme"
+        signature:
+          $ref: "#/components/schemas/Signature"
+
+    OrderReference:
```

Comment:
I renamed this here because it is slightly different than the concept of "OrderFulfillment". Specifically, it can either be:
- A reference to an auction order
- A JIT order

Note that there is no execution (or "fulfillment") information here.

Reply:
`Trade` is a term with meaning to the smart contracts (it is what the order fulfillment concept is called).

Now that I think about it, `OrderReference` is also a bad name here because a `JitOrder` is not a reference to an order, but an order itself.

`ExecutedOrder` or `TradedOrder` may make sense in this context. WDYT?

Reply:
Got rid of the name, so resolving this discussion.